public/index.html

```diff
@@ -2,14 +2,12 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
Were these just broken or unused?

Reply:
unused. I deleted the ico files and image files.